content/generators/geometric.md

```diff
@@ -403,45 +371,36 @@ The default weights for TRGG are drawn from an exponential distribution with
 rate parameter `lambda=1`.
 
 ```{code-cell} ipython3
----
-jupyter:
-  outputs_hidden: false
----
-fig, ax = plt.subplots()
-#default TRGG weight distribution network
-TRGG = nx.thresholded_random_geometric_graph(nodes,0.1,0.0001,pos=pos,weight=weight)
-nx.draw_networkx_nodes(G, pos=pos, ax=ax, **node_opts)
-draw_edges_fast(G, pos=pos, ax=ax, **edge_opts)
-```
-
-```{code-cell} ipython3
----
-jupyter:
-  outputs_hidden: false
----
-fig, ax = plt.subplots()
-#Increased threshold parameter, theta, reduces graph connectivity
-TRGG = nx.thresholded_random_geometric_graph(nodes,0.1,0.001,pos=pos,weight=weight)
-nx.draw_networkx_nodes(G, pos=pos, ax=ax, **node_opts)
-draw_edges_fast(G, pos=pos, ax=ax, **edge_opts)
+fig, axes = plt.subplots(1, 2)
+
+# Increased threshold parameter, theta, reduces graph connectivity
+thresholds = (0.0001, 0.001)
+for thresh, ax in zip(thresholds, axes):
+    TRGG = nx.thresholded_random_geometric_graph(
+        nodes, 0.1, thresh, pos=pos, weight=weight
+    )
+    nx.draw_networkx_nodes(G, pos=pos, ax=ax, **node_opts)
+    draw_edges_fast(TRGG, pos=pos, ax=ax, **edge_opts)
+    ax.set_title(f"Threshold = {thresh}, {TRGG.number_of_edges()} edges")
+fig.tight_layout()
 ```
 
 ## References
 
-[1] - Spatial Networks https://arxiv.org/pdf/1010.0302.pdf
+[^1]: Spatial Networks <https://arxiv.org/abs/1010.0302>
 
-[2] - Random Geometric Graphs https://arxiv.org/pdf/cond-mat/0203026.pdf
+[^2]: Random Geometric Graphs <https://arxiv.org/abs/cond-mat/0203026>
 
-[3] - Unit Disk Graphs http://www.sciencedirect.com/science/article/pii/0012365X9090358O
+[^3]: Unit Disk Graphs <http://www.sciencedirect.com/science/article/pii/0012365X9090358O>
 
-[4] - Waxman Graphs - http://ieeexplore.ieee.org/document/12889/
+[^4]: Waxman Graphs <http://ieeexplore.ieee.org/document/12889/>
 
-[5] - Waxman Graph Parameter Estimation - https://arxiv.org/pdf/1506.07974.pdf
+[5]: Waxman Graph Parameter Estimation <https://arxiv.org/abs/1506.07974>
```

Comment:
This looks like a dangling link, it's not referenced anywhere.

Reply:
Yup - I left it in just in case it was intended to be linked somewhere, but I think it makes more sense just to remove it (4463cf7)